interface/src/components/upload1.js

```diff
@@ -43,17 +44,21 @@ class Upload extends Component{
                 console.log(key);
                await parseToCsv(this.state[key].url)
                 .then((data)=>{
-                    console.log(data)
-                    this.setState({[key]:data})
+                     str.pop();
+                     this.setState({[key]:data})
                 })
-                .catch(err=> console.log(err))
+                .catch(err=> err_link.push(key))
             }
         }
         if(str.length===0){
             this.props.getFiles(this.state)
             return this.props.history.push("/upload2");
         }
-        str.join(", ");
+        if(err_link.length!==0){
+             err_link=err_link.join(", ")
+             alert(`Please check your google sheet link for ${err_link} files`);
+        }
+        str= str.join(", ");
         alert(`You have not uploaded your ${str} files`);
```

Comment:
```suggestion
        alert(`Please uploaded your ${str} files before continuing`);
```
This is another spot where an error component may be handy instead of an alert

Reply:
Again, there is a component for errors ([interface/src/components/error.js](https://github.com/PecanProject/BETYdb-YABA/blob/688fcfe3f7e08535013586c06d0689ccb0f4d114/interface/src/components/error.js)) but we just want to overlay a message for not uploading some files rather redirecting to an error page